aiida/cmdline/commands/cmd_calcjob.py

```diff
@@ -90,23 +90,31 @@ def calcjob_inputcat(calcjob, path):
     """
     from aiida.plugins.entry_point import get_entry_point_from_class
 
+    # Get path from the given CalcJobNode
     if path is None:
-
         path = calcjob.get_option('input_filename')
```

Comment:
what about replacing this with
`path = calcjob.get_option('input_filename') or 'aiida.in'`
and getting rid of all this extra logic?

Reply:
Good idea.